mne/stats/regression.py

```diff
@@ -134,3 +143,208 @@ def _fit_lm(data, design_matrix, names):
         mlog10_p_val[predictor] = -np.log10(p_val[predictor])
 
     return beta, stderr, t_val, p_val, mlog10_p_val
+
+
+def linear_regression_raw(raw, events, event_id=None,
+                          tmin=-.1, tmax=1,
+                          covariates=None,
+                          reject=True, tstep=1.,
```

Comment:
With a dict - True means a hardcoded default dict is called cause I'm lazy :)






On Mon, Jul 20, 2015 at 9:31 AM -0700, "Eric Larson" <notifications@github.com> wrote:












In mne/stats/regression.py:
> @@ -134,3 +143,208 @@ def _fit_lm(data, design_matrix, names):
>          mlog10_p_val[predictor] = -np.log10(p_val[predictor])
>  
>      return beta, stderr, t_val, p_val, mlog10_p_val
> +
> +
> +def linear_regression_raw(raw, events, event_id=None,
> +                          tmin=-.1, tmax=1,
> +                          covariates=None,
> +                          reject=True, tstep=1.,



I don't think reject=True is our standard way of doing this -- how do we do it elsewhere?



—
Reply to this email directly or view it on GitHub.